pkg/aws/ec2/ec2.go

```diff
@@ -203,6 +206,17 @@ func parseENI(iface *ec2_types.NetworkInterface, vpcs ipamTypes.VirtualNetworkMa
 		}
 	}
 
+	for _, prefix := range iface.Ipv4Prefixes {
+		ips, e := ipPkg.PrefixToIps(aws.ToString(prefix.Ipv4Prefix))
+		err = e
+		if err != nil {
+			err = fmt.Errorf("unable to parse CIDR %s", aws.ToString(prefix.Ipv4Prefix))
```

Comment:
do you think original error (e.g. e) might have some useful detail ?

Reply:
It might be helpful. Included error from `PrefixToIps` in error message.